.github/workflows/build.yml

```diff
@@ -4,9 +4,13 @@ on:
   push:
     branches: [ main ]
     tags:
-      - "*"
+      - '*'
   pull_request:
-    branches: [main, "release/*"]
+    branches: [main, 'release/*']
+
+env:
+  STABLE_TAG: ${{ github.event_name == 'push' && github.ref_name || format('pr-{0}', github.event.pull_request.number) }}
+  EXPIRATION_LABEL: ${{ github.event_name == 'push' && 'quipucords.source=github' || 'quay.expires-after=3d' }}
```

Comment:
I had trouble with GitHub setting quay expiration label if I merged to main. No matter if I skipped condition, set variable to empty string or null. So I just put any meanigless value as a workaround.

Reply:
I think it's worth commenting this on the code (I will certainly forget this haha)

Reply:
Done in 21f90b7ed31d706668f7727c0830ca9419da2900